package/src/frontend/browser/Agent.js

```diff
@@ -97,6 +142,26 @@ export const Agent = function(userAgent) {
     var match = userAgent.match(exp);
     return match && match[1] && parseInt(match[1], 10) >= version;
   }
+
+  function matchesMinVersion(exp, version) {
+    var match = userAgent.match(exp);
+    //Chrome
+    if (match[0].includes('Chrome')) {
+      return match && match[1] && parseInt(match[1], 10) >= version;
+    }
+    //Edge
+    if (match[0].includes('Edg')) {
+      return match && match[1] && parseInt(match[1], 10) >= version;
+    }
+    //Safari
+    if (match[0].includes('Version')) {
+      return match && match[1] && parseInt(match[1], 10) >= version;
+    }
+    //Firefox
+    if (match[0].includes('Firefox')) {
+      return match && match[1] && parseInt(match[1], 10) >= version;
+    }    
+  }
```

Comment:
All these cases do the same.

Reply:
It does, and basically it's the same as the existing `captureGroupGreaterOrEqual` function. 
So I will replace that.